components/cards/cards.css

```diff
@@ -0,0 +1,100 @@
+.card_container {
+    position: relative;
+    background-color: var(--gray5);
+    box-sizing: border-box;
+    padding: 1rem;
+    border-radius: var(--br-sq);
+}
+
+.card_container:hover {
+    box-shadow: 0 3px 20px 0 rgba(0, 0, 0, 0.5);
+}
+
+.card-vr {
+    display: flex;
```

Comment:
some places you have used dash & at some places you have used underscore.
please check & update that 
ex : card_container in above class & here it is card-vr

Reply:
Yes that is intentional. - is used for variant of the component. _ is used for section in the code for component